iot_config/src/route.rs

```diff
@@ -543,49 +547,63 @@ pub fn active_route_stream<'a>(
             active: route.active,
             locked: route.locked,
             ignore_empty_skf: route.ignore_empty_skf,
-        })})
+        }, deleted))})
```

Comment:
rename `filter_map(|route| async move { route.ok() } )` to `filter_map(|route_tuple| async move { route_tuple.ok() } )`

Reply:
Address in 1390ffc